### ISSUE TITLE
'Vanilla' Lua compatibility

### ISSUE THREAD

**Issue by SquidDev** [TARDIX/Kernel#9]
Sorry to post another issue (and again I'm not sure if this fits in with the aims of this project - though I love what you've got so far!) but a reimplementation of some of Lua's default libraries would be great. I know this is similar to #8 but addresses a wider range of things.

I guess this would include:

 - `require`/`package`. 
 - `io` (`seek` wouldn't be feasible without reading the entire file as a string which isn't really desirable). Though streams would mean `popen` would be possible.
 - `os`: `getenv` would be possible I guess?

I [wrote some stuff](https://github.com/SquidDev-CC/Blue-Shiny-Rocks/tree/master/env) ages ago if you want to adapt it. Again I am happy to setup a PR, just would like some thoughts.

**Comment by DemHydraz** [comment]
Yes, it does fit with the goals of the project. And, require/package are partially implemented (not sure if `run.require` fits your use cases - it loads files under /usr/lib, much like os.loadAPI, but not using it), io and getenv() are next on our list. 

io will be possible as soon as we get the virtual file system working properly.

If you wish to discuss this stuff more, it'd be good if we could talk on the gitter room.

Thank you,
DemHydraz
